resource_activity/views/resource_activity_views.xml

```diff
@@ -79,6 +79,7 @@
                     <header>
                         <button name="create_sale_order" string="Create Quotation" type="object" states="draft"/>
                         <button name="action_quotation" string="Set to Quotation" type="object" states="sale"/>
+                        <button name="print_last_sale_order" string="Print Last Quotation" type="object" states="quotation"/>

```

Comment:
Il me semble que ce bouton devrait être accessible depuis les états `sale` et `done`. cf tâche https://gestion.coopiteasy.be/web#id=1336&view_type=form&model=project.task